<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeEditorWindow</class>
 <widget class="QDialog" name="NodeEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>30</width>
    <height>2</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1280</width>
    <height>780</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Render Graph Editor</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
  </layout>
  <action name="actionCreateUserDefinedNode">
   <property name="text">
    <string>Create user-defined Node</string>
   </property>
   <property name="toolTip">
    <string>Create a node from a description in data/nodes</string>
   </property>
  </action>
  <action name="actionRemoveNode">
   <property name="text">
    <string>Remove Node</string>
   </property>
   <property name="toolTip">
    <string>Remove the node from the Graph</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the current graph</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCreateTextureNode">
   <property name="text">
    <string>Create Texture Node</string>
   </property>
   <property name="toolTip">
    <string>Create a texture node</string>
   </property>
  </action>
  <action name="actionCreateFloatNode">
   <property name="text">
    <string>Create Float Node</string>
   </property>
  </action>
  <action name="actionCreateAdditionNode">
   <property name="text">
    <string>Create Addition Node</string>
   </property>
  </action>
  <action name="actionCreateSubtractionNode">
   <property name="text">
    <string>Create Subtraction Node</string>
   </property>
  </action>
  <action name="actionCreateMultiplicationNode">
   <property name="text">
    <string>Create Multiplication Node</string>
   </property>
  </action>
  <action name="actionCreateDivisionNode">
   <property name="text">
    <string>Create Division Node</string>
   </property>
  </action>
  <action name="actionCreateGreaterThanNode">
   <property name="text">
    <string>Create GreaterThan Node</string>
   </property>
  </action>
  <action name="actionCreateEqualToNode">
   <property name="text">
    <string>Create EqualTo Node</string>
   </property>
  </action>
  <action name="actionCreateNotEqualToNode">
   <property name="text">
    <string>Create NotEqualTo Node</string>
   </property>
  </action>
  <action name="actionCreateLessThanNode">
   <property name="text">
    <string>Create LessThan Node</string>
   </property>
  </action>
  <action name="actionCreateGreaterThanOrEqualToNode">
   <property name="text">
    <string>Create GreaterThanOrEqualTo Node</string>
   </property>
   <property name="toolTip">
    <string>Create GreaterThanOrEqualTo Node</string>
   </property>
  </action>
  <action name="actionCreateLessThanOrEqualToNode">
   <property name="text">
    <string>Create LessThanOrEqualTo Node</string>
   </property>
   <property name="toolTip">
    <string>Create LessThanOrEqualTo Node</string>
   </property>
  </action>
  <action name="actionCreateNegationNode">
   <property name="text">
    <string>Create Negation Node</string>
   </property>
  </action>
  <action name="actionCreateAbsoluteNode">
   <property name="text">
    <string>Create Absolute Node</string>
   </property>
  </action>
  <action name="actionCreateNotNode">
   <property name="text">
    <string>Create Not Node</string>
   </property>
  </action>
  <action name="actionCreateAndNode">
   <property name="text">
    <string>Create And Node</string>
   </property>
   <property name="toolTip">
    <string>Create And Node</string>
   </property>
  </action>
  <action name="actionCreateOrNode">
   <property name="text">
    <string>Create Or Node</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
